### DISCUSSION TITLE
Two instances slinger same box same time?

### DISCUSSION THREAD

**Discussion by Toei79** [GerryDazoo/Slinger#383] [General]
Hi its possible do this?
My goal its use component  , and composite at the same time [slinger in different  folders]
Or maybe automatize the use so just use links for use component  or composite without the need change manually .

I guess it guess need be same time at least script running so I don't need add some manually

**Comment by GerryDazoo** [comment]
No it's not possible. The slingbox hardware/software does not support it.

**Reply by Toei79** [reply]
> No it's not possible. The slingbox hardware/software does not support it.

what about this, run two instances of slinger same time , one for component , one for composite but just use one then switch to the other one , but not in the same , just to automatize and for not use manual switch component - composite in my case.

**Reply by GerryDazoo** [reply]
Yes you can do that, But don't run two copies of the server. It supports passing in multiple config.ini files but I would recommend using the unified_config.ini example. Read the release notes.

**Reply by Toei79** [reply]
ok im going to read that. so its possible thank you

**Reply by Toei79** [reply]
> Yes you can do that, But don't run two copies of the server. It supports passing in multiple config.ini files but I would recommend using the unified_config.ini example. Read the release notes.

so this part there can be like this


[SLINGBOXES]
sb1=350
sb2=350
sb3=M1



im checking tonight , and yea , i see, different ports also different sources. 

thanks!

**Reply by Toei79** [reply]
> Yes you can do that, But don't run two copies of the server. It supports passing in multiple config.ini files but I would recommend using the unified_config.ini example. Read the release notes.

Edit

finally i able to run it, 
so 
sb1= , sb2= , can be any name and same details info for one slingbox. (it cant be same name , it need be different sb= ,)  after trial in error  and read the notes. i figure out, 
let this one just in case , someone needs it. 
* running different video sources from the same slingbox. [two different stream links for this one] in my case 350

> ; This a sample of a unified config.ini for three slingboxes

[SLINGBOXES]
sb1=350
;sb2=Solo
sb3=450

[350]
sbtype="350/500/M1"
password=8aUh0plqPhSiQAx
;finderid=nogood4CF70531865D73650A16A0A536
;ipaddress=192.168.2.132
;port=5201
;Resolution=12
;FrameRate=60
;VideoBandwidth=2000
;VideoSmoothness=63
VideoSource=1
;maxstreams=3
;StartChannel=155
;RemoteLock=yes
;Remote=500_remote.txt

;[Solo]
;sbtype="Solo/Pro/ProHD"
;password=AnotherPassword     
;ipaddress=192.168.117.101
;finderid=BAD95DAED609364C8204F2BC5FBDC363
;port=5001
;Resolution=5
;FrameRate=30
;VideoBandwidth=2000
;VideoSmoothness=63
;VideoSource=2
;RemoteLock=yes
;StartChannel=155
;Remote=solo_remote.txt

[450]
sbtype="350/500/M1"
password=8aUh0plqPhSiQAx
;finderid=E706A7405882C03A076502246DAD632F     
;ipaddress=192.168.2.132
;port=5301
;Resolution=12
;FrameRate=30
;VideoBandwidth=2000
;VideoSmoothness=63
VideoSource=0
;Remote=m1_remote.txt

[SERVER]
; local port number for the server to listen on for connections
; Send all streaming and Remote request to this port number.
port=8080
maxremotestreams=8
URLbase=Gerry
;enableremote=yes

>

**Reply by GerryDazoo** [reply]
Yes, this is OK. You can as many different configurations you want of a single slingbox. THe caevaet being you can only use one at a time.
You may want to change your URLbase to something besides Gerry, unless you really like typing my name.

**Reply by Toei79** [reply]
> Yes, this is OK. You can as many different configurations you want of a single slingbox. THe caevaet being you can only use one at a time. You may want to change your URLbase to something besides Gerry, unless you really like typing my name.

lol yea thats fine , but its very cool feature too , give a name to the stream link.  

question those boxes only have one audio so it you play component or composite you will probably hear the others audio input. 

its anyway to split  rca channels,  i mean just hear left for one and right for the other  video source.

so my guessing it was designed like that for save bandwith with composite (350 model) otherwise i dont get it why both component and composite share audio inputs.
